### PR TITLE
Correcting field name conventions

### PR DESCRIPTION
**Pull request by mb010**
This [comment](https://github.com/MultimodalUniverse/MultimodalUniverse/issues/123#issuecomment-2387928966) in #123 highlights some naming conventions and irregularities across datasets. This PR aims to address those that need to be corrected acording to our schema of shared features (largely datset names, redshift, ra, dec, with most other metadata being allowed in survey specific formats).

### CODE REVIEW

**Comment by mb010** [comment]
- [ ] `array` should be `flux` in imaging datasets where appropriate.

File: scripts/desi/desi.py [hunk 1]
```diff
@@ -51,21 +51,21 @@
...
+    "flux_z",
+    "flux_ivar_g",
+    "flux_ivar_r",
+    "flux_ivar_z",
+    "fiberflux_g",
+    "fiberflux_r",
+    "fiberflux_z",
+    "fibertotflux_g",
+    "fibertotflux_r",
+    "fibertotflux_z",
```
Comment (mb010) [inline scripts/desi/desi.py:68]:
```suggestion
    "ZERR",
    "EBV",
    "FLUX_G",
    "FLUX_R",
    "FLUX_Z",
    "FLUX_IVAR_G",
    "FLUX_IVAR_R",
    "FLUX_IVAR_Z",
    "FIBERFLUX_G",
    "FIBERFLUX_R",
    "FIBERFLUX_Z",
    "FIBERTOTFLUX_G",
    "FIBERTOTFLUX_R",
    "FIBERTOTFLUX_Z",
```

File: scripts/vipers/build_parent_sample.py [hunk 2]
```diff
@@ -78,6 +78,8 @@ def extract_data(filename):
     # Loop through the header keys and add them to the results dictionary
     for key in HEADER_KEYS:
         results[key] = float(header[key])
+        if key == "REDSHIFT":
+            results["z"] = float(header[key])
```
Comment (LTMeyer) [inline scripts/vipers/build_parent_sample.py:82]: This way we still have `result["REDSHIFT"]` as an entry. If it is not used nor wanted I suggest the following change.
```suggestion
        if key == "REDSHIFT":
            results["z"] = float(header[key])
        else:
            results[key] = float(header[key])
```

**Review by LTMeyer — APPROVED** [review]
One minor comment otherwise it looks good.

**Pull request merged**